x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/step_define_form.test.tsx

```diff
@@ -76,7 +76,8 @@ describe('Transform: <DefinePivotForm />', () => {
 
     // Act
     // Assert
-    expect(getByLabelText('Index pattern')).toBeInTheDocument();
```

Comment:
Here it threw error that `Index pattern` label is labelling `span`, which is incorrect. 
I fixed the test, but this component should be reviewed from labels/a11y perspective 

This was the error here:

```
TestingLibraryElementError: Found a label with the text of: Index pattern, however the element associated with this label (<span />) is non-labellable [https://html.spec.whatwg.org/multipage/forms.html#category-label]. If you really need to label a <span />, you can use aria-label or aria-labelledby instead.
```

Reply:
Thanks for flagging this up @Dosant . Yes we have a `<span>` inside an `<EuiFormRow>` on this page. Do you know what would be better from the a11y perspective? Should we replace the `<span>` with `EuiText`? This form row is displaying a non-editable piece of text (here 'gallery-*' is the span content):

![image](https://user-images.githubusercontent.com/7405507/94122229-36e16e80-fe4a-11ea-9c33-97bf02d362c8.png)